nasa/harmony-regression-tests
# DAS-1701 - Remove leading slashes from requested variable names.

## Description

This PR accommodates recent changes to the UMM-Var records for collections hosted in the EEDTEST UAT provider. These collections have been updated to remove leading slashes from the UMM-Var `Name` attributes (e.g., `/BEAM0000/lat_lowestmode` has become `BEAM0000/lat_lowestmode`) **only for top-level variables**.

Update:

There has been a bit of back-and-forth on this in Slack. For the purposes of this ticket, I've _only_ updated top-level variables to remove the slash. This includes the MERRA-2 collections (M2T1NXSLV and M2I3NPASM), one of which is used in a test in this notebook. If it is determined, that all variable need to be updated, I'll conduct that work in a separate ticket, as this may have wider reaching implications for Harmony itself.

## Jira Issue ID

DAS-1701

## Local Test Steps

* Pull this branch of the repository.
* Start a Jupyter notebook server.
* Update the `harmony_host_url` in the first cell to point to SIT (SIT has a more up-to-date version of MaskFill, which passes a test. UAT doesn't have this version, yet).
* Run the tests.
* All should pass.

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
